src/app/files/file-explorer.js

```diff
@@ -228,7 +234,9 @@ function fileExplorer (localRegistry, files, menuItems) {
       }
       actions['Delete'] = () => {
         if (self.files.isReadOnly(key)) { return tooltip('cannot delete folder. ' + self.files.type + ' is a read only explorer') }
-        modalDialogCustom.confirm('Confirm to delete a folder', 'Are you sure you want to delete this folder?',
+        const currentFilename = extractNameFromKey(key)
+
+        modalDialogCustom.confirm(`Confirm to delete a folder', 'Are you sure you want to delete ${currentFilename} folder?`,
```

Comment:
Could you also add the name of the file to the title?
`Confirm to delete "${currentFilename}" folder'

Reply:
Done.